tests/input/examples/schema_definition-native-array-2.yaml

```diff
@@ -0,0 +1,127 @@
+id: https://example.org/arrays
+name: arrays-temperature-example-2
+title: Array Temperature Example Using NDArray Classes
+description: |-
+  Example LinkML schema to demonstrate a 3D DataArray of temperature values with labeled axes
+  using LinkML NDArray classes
+license: MIT
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  wgs84: http://www.w3.org/2003/01/geo/wgs84_pos#
+  example: https://example.org/
+
+default_prefix: example
+
+imports:
+  - linkml:types
+
+classes:
+
+  TemperatureDataset:
+    tree_root: true
+    implements:
+      - linkml:DataArray
+    annotations:
+      array_data_mapping:
+        data: temperatures_in_K
+        dims: [x, "y", t]
+        coords:
+          latitude_in_deg: x
+          longitude_in_deg: "y"
+          time_in_d: t
+    attributes:
+      name:
+        identifier: true
+        range: string
+      latitude_in_deg:
+        implements:
+          - linkml:axis
+        range: LatitudeSeries
+        required: true
+        annotations:
+          axis_index: 0
+      longitude_in_deg:
+        implements:
+          - linkml:axis
+        range: LongitudeSeries
+        required: true
+        annotations:
+          axis_index: 1
+      time_in_d:
+        implements:
+          - linkml:axis
+        range: DaySeries
+        required: true
+        annotations:
+          axis_index: 2
+      temperatures_in_K:
+        implements:
+          - linkml:array
```

Comment:
Should we rename this in the metamodel? Is this scoped to `linkml:DataArray` or unique across LinkML?